### PR TITLE
 Add test to consume content using mazer

### PR DESCRIPTION
**Pull request by kersommoura**
Add test to consume content hosted by Pulp using mazer.

closes:#4915

Required PR:PulpQE/pulp-smash#1212

### CODE REVIEW

**Comment by daviddavis** [comment]
This LGTM. @bmbouter ?

**Comment by kersommoura** [comment]
The failing test is passing locally. 

``` bash
 pulp_ansible/tests/functional/api/collection/test_consume_content.py::MazerConsumeCotentTestCase.test_consume_content ✓100% ██████████

Results (6.34s):
       1 passed
```

Locally the extra variables were added to the the systemd env.
```
Environment="PULP_ANSIBLE_API_HOSTNAME=http://localhost:24817" Environment="PULP_ANSIBLE_CONTENT_HOSTNAME=http://localhost:24816/pulp/content"
```

**Comment by daviddavis** [comment]
@kersommoura you could set them in the travis environment by setting them in `.travis/post_before_script.sh`

**Comment by kersommoura** [comment]
@daviddavis, thanks.

Somehow I did think that this [PR](https://github.com/pulp/pulp_ansible/commit/3c3eb5d5dc32a14af739bfe7d6baca89538f71f7) added the same settings, in a different manner.

**Comment by daviddavis** [comment]
Theoretically we should be able to update the settings in [the Travis playbook](https://github.com/pulp/pulp_ansible/blob/master/.travis/playbook.yml) but it's managed by the plugin_template and there's no way to do it currently.

**Comment by bmbouter** [comment]
@kersommoura pulp_ansible needs to have these variables set all the time so I believe the right place is to set them in here with this PR https://github.com/pulp/pulp_ansible/blob/master/.travis/playbook.yml#L26

File: test_requirements.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ git+https://github.com/PulpQE/pulp-smash.git#egg=pulp-smash
 # pin pydocstyle until https://gitlab.com/pycqa/flake8-docstrings/issues/36 is resolved
 pydocstyle<4
 pytest
+mazer
```
Comment (daviddavis) [inline test_requirements.txt]: Is this needed? I feel like in a test environment, we'd install the packages from setup.py and mazer is there already:

https://github.com/pulp/pulp_ansible/blob/master/setup.py#L5

Comment (kersommoura) [reply test_requirements.txt]: You are right. I removed the other one.

**Pull request merged**